example/src/screens/Home.tsx

```diff
@@ -113,6 +113,20 @@ const Home = () => {
 
   const onShowPrePurchaseVenue = async () => {
     try {
+      const onEDPSelectionStarted = () => {
+        console.log('do stuff here');
+      };
+      const onMenuItemSelected = () => {
+        console.log('do stuff here');
+      };
+      const openURLNotSupported = () => {
+        console.log('do stuff here');
+      };
+      await RetailSDK.setupUserAnalytics(
+        onEDPSelectionStarted,
+        onMenuItemSelected,
+        openURLNotSupported
+      );
```

Comment:
I guess the TODO logs are because we haven't decided what we'd like to do here? I guess we could change the logs to more descriptive for now anyway, eg. "in menu item selected" etc.